src/DOM/HTML/Event/DragEvent/DataTransfer.purs

```diff
@@ -25,3 +29,23 @@ foreign import filesNullable :: DataTransfer -> Nullable FileList
 -- | Returns an array of data formats used in the drag operation.
 -- | If the drag operation included no data, then the array is empty.
 foreign import types :: DataTransfer -> Array String
+
+-- | Retrieves the data for a given format, or an empty string if data for that
+-- | format does not exist or the data transfer object contains no data.
+foreign import getData
+  :: forall eff
+   . String
```

Comment:
How about `MediaType` from purescript-media-types? It's already used in DOM.File.{Blob,File}.